### DISCUSSION TITLE
Remove margin between carousel images

### DISCUSSION THREAD

**Discussion by briltec** [community/community#45102] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Why is there a vertical margin line between images in this Flutter carousel? The line is flickering when I slide the carousel.
`import 'package:carousel_slider/carousel_slider.dart';
import 'package:flutter/material.dart';

main() {
  runApp(
    MaterialApp(
      home: Scaffold(
        appBar: AppBar(),
        body: ListView(
          children: [
            Text('Test'),
            CarouselSlider(
              options: CarouselOptions(
                aspectRatio: 1.8,
              ),
              items: List.generate(5, ((i) {
                return Container(
                  color: Colors.grey[200],
                  child: Column(
                    children: [
                      Image.network(
                        'http://picsum.photos/id/$i/400/200',
                      ),
                      Text('TEST'),
                    ],
                  ),
                );
              })),
            ),
          ],
        ),
      ),
    ),
  );
}`

The code which is commented will result in the right behaviour but without carousel. As soon i try it with a listview following a carouselslider i get the overflow issue as above. I also tried adding "overflow: overflow.visible" but as it is depreciated there is no effect visible.

Anyone out there who can help me please.

**Comment by metabrainiac** [comment]
The issue I've got from container border. So if you use container on top widget it will have padding and will get this. It is easier to deal with Stack widget in this case.
`CarouselSlider(
  options: CarouselOptions(
    aspectRatio: 1.8,
    viewportFraction: .4,
    clipBehavior: Clip.none,
  ),
  items: List.generate(5, ((i) {
    return itemBUild(i);
  })),
),
And item build

Widget itemBUild(int id) {
  return Stack(
    fit: StackFit.expand,
    children: [
      // Image.network(
      //   'http://picsum.photos/id/$i/400/200',
      // ),
      Positioned.fill(
        child: Container(
          decoration: BoxDecoration(
              color: Colors.red, border: Border.all(color: Colors.red)),
        ),
      ),
      Positioned(
        bottom: 0,
        left: 0,
        right: 0,
        child: Container(
          color: Colors.green,
          child: Text('TEST', textAlign: TextAlign.center,),
        ),
      ),
    ],
  );
}`

**Comment by briltec** [comment]
Unfortunately, it doesn't.

**Comment by metabrainiac** [comment]
Are you referring to the gap between images?

**Comment by briltec** [comment]
Yes, the gap should be removed.

**Comment by metabrainiac** [comment, marked as answer]
Does not work any better. Same bug except I have a red gap instead of a white gap.

**Comment by briltec** [comment]
Seems like too device specific now.

**Comment by bryancanlas** [comment]
I think you use this package: carousel_slider

Official link: https://pub.dev/packages/carousel_slider

Blog link: https://androidride.com/flutter-carousel-slider-example/

Thank You
